e2e-tests/cypress/integration/before-you-start-enforcement-notice/before-you-start-enforcement-notice-steps.js

```diff
@@ -0,0 +1,55 @@
+import {Given } from 'cypress-cucumber-preprocessor/steps';
+import { goToPage } from '../../support/go-to-page/goToPage';
+import { verifyPageHeading } from '../../support/common/verify-page-heading';
+import { verifyPageTitle } from '../../support/common/verify-page-title';
+import {
+  getEnforcementNoticeErrorMessage,
+  getEnforcementNoticeNo,
+  getEnforcementNoticeYes,
+} from '../../support/page-objects/enforcement-notice-po';
+import { getContinueButton, getErrorMessageSummary } from '../../support/page-objects/common-po';
+import { verifyErrorMessage } from '../../support/common/verify-error-message';
+const pageHeading = 'Have you received an enforcement notice?';
+const pageTitle = 'Have you received an enforcement notice? - Before you start - Appeal a householder planning decision - GOV.UK';
```

Comment:
This title is as per the ticket but is this correct? (This is a common page for Householder and other planning decision)